qiskit_ibm_runtime/options/resilience_options.py

```diff
@@ -39,21 +39,21 @@ class ResilienceOptionsV2:
 
     Args:
         measure_mitigation: Whether to enable measurement error mitigation method.
-            Further suboptions are available in :attr:`~measure_noise_learning`.
+            If you enable measurement mitigation, you can fine tune its noise learning
+            by using :attr:`~measure_noise_learning`. See :class:`MeasureNoiseLearningOptions`
+            for all measurement mitigation noise learning options.
             Default: True.
 
         measure_noise_learning: Additional measurement noise learning options.
             See :class:`MeasureNoiseLearningOptions` for all options.
 
         zne_mitigation: Whether to turn on Zero Noise Extrapolation error mitigation method.
-            Further suboptions are available in :attr:`~zne`.
```

Comment:
I thought you were going to update zne/pec with the same wording? Like 

```
If you enable ZNE, you can fine tune its options 
by using :attr:`~zne`. See :class:`ZneOptions`
for additional ZNE related options.
```

Reply:
Updated in [496598e](https://github.com/Qiskit/qiskit-ibm-runtime/pull/1807/commits/496598ed4ed01535efb3149f72b58d62825e50cb)